poker-app/components/AsyncStorageController.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
     },
 
     removeData: function(){
-        AsyncStorage.setItem('key', {});
+        let empty = [{}];
+        AsyncStorage.setItem('key', JSON.stringify(empty));
```

Comment:
why not just use "[{}]" literal instead of calling JSON.stringify? You know what the output is.

Reply:
I can only save strings in storage, so i need to use stringify to save it. I actually tried with saving it as a literal first and had an Issue with it. But It was before I added some error handling so It might work now.